### DISCUSSION TITLE
anomaly detection

### DISCUSSION THREAD

**Discussion by mahilaMoghadami** [nv-morpheus/Morpheus#1817] [Experimental]
Hello

Is there any pre-trained model for anomaly detection in Nvidia Morpheus?
if  yes, how? and what is model name?

thank you

**Comment by efajardo-nv** [comment]
@mahilaMoghadami We have two anomaly detection examples each included with pre-trained model:

PCAP: https://github.com/nv-morpheus/Morpheus/tree/v24.06.00/examples/abp_pcap_detection
NVIDIA SMI logs: https://github.com/nv-morpheus/Morpheus/tree/v24.06.00/examples/abp_nvsmi_detection
